src/internal/k8s/configmap.go

```diff
@@ -51,11 +54,14 @@ func CreateConfigmap(namespace, name string, labels map[string]string, data map[
 // DeleteConfigmap delets a confimap by name
 func DeleteConfigmap(namespace, name string) error {
 	message.Debugf("k8s.DeleteConfigmap(%s, %s)", namespace, name)
-	clientSet := getClientset()
+	clientSet, err := getClientset()
```

Comment:
clientset or clientSet?  The capitalization of getClientset would imply clientset.